### ISSUE TITLE
Training script release

### ISSUE THREAD

**Issue by hetong007** [jzhoubu/vsearch#2]
Hi,

Thanks for releasing code for this interesting work! 

Since the loss design and training strategy forms a crucial part of the paper, will there be plan to release the training scripts?

Thanks.

**Comment by jzhoubu** [comment]
Hi @hetong007 . Thank you for your interest in our work and for reaching out. We do have plans to make them available. However, we haven't set a specific timeline for this release yet. We appreciate your understanding and patience on this matter.

**Comment by jzhoubu** [comment]
Hi @hetong007 We have recently released the training scripts for this repo. If you have any further questions or need more assistance, feel free to open a new issue.
